src/server/kiwix-serve.cpp

```diff
@@ -879,26 +879,46 @@ int main(int argc, char **argv) {
   }
 
   /* Compute the Welcome HTML */
-  string welcomeBooksHtml;
+  string welcomeBooksCss = ""
+"<style>"
+".book__list { text-align: center; }"
+".book {"
+    "display: inline-block; vertical-align: bottom; margin: 10px; padding: 15px 20px; width: 300px;"
+    "border: 1px solid #ccc; border-radius: 15px;"
+    "text-align: left; color: #000; font-family: sans-serif; font-size: 13px;"
+"}"
+".book:hover { background-color: #eee; box-shadow: 2px 2px 5px 0px #ccc}"
+".book__background { background-repeat: no-repeat; background-size: auto; background-position: top right; }"
+".book__title { font-size: 18px; color: #0645ad; }"
+".book__description { padding: 5px 50px 5px 0px; font-size: 15px; }"
+".book__info { line-height: 18px; color: #777; font-weight: bold; font-size: 13px; }"
+"</style>";
+
+  string welcomeBooksHtml = ""
+"<div class='book__list'>";
```

Comment:
I did it to keep some kind of an indentation for the html.
Creating template in a backend language can quickly make code complicated. I do this to keep the html as much readable as I can.
The other bright side is that the html generated when viewing the source in the browser (right click > inspect source) has a right indentation. A minor addition though.